app/helpers/pledges_helper.rb

```diff
@@ -15,4 +15,12 @@ def cancel_pledge_button
       'Cancel pledge'
     end
   end
+
+  def delete_patient_button
+    content_tag :span, class: 'btn btn-primary btn-lg cancel-btn btn-block',
```

Comment:
I'm not sure I follow the logic on updating the view, but I'd definitely like to update it with some guidance! Otherwise, moving relevant method to the `patients_helper`